packages/react-tinacms-github/README.md

```diff
@@ -16,19 +36,22 @@ const exitEditMode = () => {
     window.location.reload()
   })
 }
-const YourLayout = ({ Component, pageProps }) => {
+const YourLayout = ({ Component, pageProps, children }) => {
   return (<TinacmsGithubProvider
-      authenticate={() => authenticate('/api/create-github-access-token')}
+      authenticate={() => authenticate(process.env.GITHUB_CLIENT_ID, '/api/create-github-access-token')}
```

Comment:
Unrelated to docs, but we should adjust the API so `authenticate` doesn't need to be imported?

```
import { TinacmsGithubProvider } from 'react-tinacms-github'

//...

<TinacmsGithubProvider
   githubClientId={process.env.GITHUB_CLIENT_ID}
   authCallbackRoute='/api/create-github-access-token'
   enterEditMode={enterEditMode}>
   exitEditMode={exitEditMode}>
```

Reply:
Also I'm getting an compiler warning by passing the client id in as the first arg. 🤷‍♂

Reply:
The clientId as the first param is changed along in this PR: https://github.com/tinacms/tinacms/pull/986/files#diff-ec898f6d33b5511d87ec653cbcbc63b4R21

Reply:
But yeah agreed on moving authenticate inside TinacmsGithubProvider, I'll make a note to pull that over in a subsequent PR